src/typescript-worker/util.ts

```diff
@@ -118,3 +118,15 @@ export const parseNpmStyleSpecifier = (
   ];
   return {pkg, version: version ?? '', path};
 };
+
+/**
+ * Return the file extension of the given URL path. Does not include the leading
+ * ".".
+ */
+export const fileExtension = (path: string): string => {
```

Comment:
Not sure if this is out of scope for this but does this need to handle declaration extension, i.e. `test.d.ts` returns `.ts` currently?